ProfileView/src/[sandbox]/sandbox.ts

```diff
@@ -151,7 +150,7 @@ export class Sandbox extends LitElement {
           >
             <cjaas-profile-view-widget
               template-id="second-template"
-              customer="30313-Carl"
+              .customer=${undefined}
```

Comment:
why is this undefined?

Reply:
I was checking a test case for undefined customer Id.